docs/en/advanced/airspeed_validation.md

```diff
@@ -0,0 +1,87 @@
+# Airspeed Validation
+
+PX4 includes a set of airspeed validation checks that run continuously during fixed-wing flight to ensure the airspeed data is accurate and reliable.
+If any check fails persistently, the system can declare airspeed as invalid.
```

Comment:
What happens if a check is invalid? I'd say something like "The sensor data may then be discarded, with a fallback to other sensors or estimates". If all sensor data is lost Xxxx